iDiff/differs/aptDiff.go

```diff
@@ -0,0 +1,94 @@
+package differs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// AptDiff compares the packages installed by apt-get.
+func AptDiff(img1, img2 string) (string, error) {
+	pack1, err := getPackages(img1)
+	if err != nil {
+		return "", err
+	}
+	pack2, err := getPackages(img2)
+	if err != nil {
+		return "", err
+	}
+
+	diff1, diff2 := diffMaps(pack1, pack2)
+	s1 := fmt.Sprintf("Image %s had the following packages which differed:\n%s\n", img1, strings.Join(diff1, "\n"))
+	s2 := fmt.Sprintf("\nImage %s had the following packages which differed:\n%s", img2, strings.Join(diff2, "\n"))
+	return s1 + s2, nil
```

Comment:
this is a pretty good use case for golang [templating](https://golang.org/pkg/text/template/): you can check out [this PR](https://github.com/GoogleCloudPlatform/runtimes-common/pull/140/) for an example of how to use it. at the very least, we should combine these strings using a multiline string.